pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -158,7 +158,7 @@ def cache_training_step_metrics(self, opt_closure_result):
         self.logged_metrics.update(logged_metrics_tmp)
         self.cached_results.legacy_batch_log_metrics.update(logged_metrics_tmp)
 
-    def log_metrics(self, metrics, grad_norm_dic, step=None, log_train_step_metrics=False):
+    def log_metrics(self, metrics, grad_norm_dic, step=None):
```

Comment:
is it used by a user, right? then let's hold back compatibility with API and add a warning...

Reply:
It is internal. LoggerConnector class

Reply:
Not it is not.